extensions/Worker.Extensions.Http.AspNetCore/src/FunctionsMiddleware/FunctionsHttpProxyingMiddleware.cs

```diff
@@ -53,7 +53,10 @@ public async Task Invoke(FunctionContext context, FunctionExecutionDelegate next
             {
                 ActionContext actionContext = new ActionContext(httpContext, httpContext.GetRouteData(), new ActionDescriptor());
 
-                await actionResult.ExecuteResultAsync(actionContext);
+                if (!httpContext.RequestAborted.IsCancellationRequested && !context.CancellationToken.IsCancellationRequested)
```

Comment:
Can we add a comment that this is a mitigation and the dispose exception is still technically possible?